spec.yaml

```diff
@@ -601,6 +615,28 @@ paths:
         omitted, default configuration values will be applied. For example, if
         no `release_version` is specified, the most recent version is used.
 
+        The number of `availability_zones` affects the total number of nodes
+        that can be created in the cluster. The number of availability zones
+        splits the ip range that can be used for the cluster in multiple smaller
+        ip ranges. The [getInfo](#operation/getInfo) endpoint provides more
+        details about the cluster ip range.
+
+        IP range example:
+
+        If a cluster gets a `/22` (1022 hosts). And the cluster should be
+        spawned across 3 availability zones. The range will then be split up
+        into four `/24` (254 hosts) that can be given to four different
+        availability zones. One range will be stay unused because ranges can
+        only be split binary. Each of the `/24` will then be split into two
+        `/25` (126 hosts) for public and private subnets. The private subnet is
+        used for nodes and internal loadbalancer (only if you create them
+        within Kubernetes). The public subnet will be used by the public
+        loadbalancers. Your cluster comes with two public loadbalancers already.
+        One for the Kubernetes API and one for Ingress.
+
+        __Note:__ AWS ELBs can take up to 8 IP addresses due to the way how
+        they scale.
```

Comment:
`they scale. In addition to this, every AWS subnet has four first addresses (.1-.4) reserved for internal use.`